ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OWord2VecTokenizerParams.scala

```diff
@@ -16,32 +16,40 @@
  */
 package ai.h2o.sparkling.ml.params
 
-import org.apache.spark.ml.param.{Param, Params, StringArrayParam}
+import org.apache.spark.ml.param.{IntParam, Param, Params, StringArrayParam}
 
 trait H2OWord2VecTokenizerParams extends Params {
 
   //
   // Param definitions
   //
   private final val stopWords = new StringArrayParam(this, "stopWords", "List of stop words.")
-  private final val inputCol = new Param[String](this, "inputCol", "input column name")
+  private final val inputCol = new NullableStringParam(this, "inputCol", "input column name")
```

Comment:
Does it make sense to allow `null` values on `inputCol`? if `inputCol` is null what is the subject of tokenization?

Reply:
If we don't allow nulls, then we need to set input column to some default value which I wated to avoid

Reply:
But happy to change it if you think this is not a good pattern

Reply:
IMHO, we should follow as it's in Spark mllib. People are used to the Spark's behavior if the input column is not set. I'm afraid `NullPointerException` or similar error might be a bit misleading.

Reply:
done